src/main/kotlin/org/rust/lang/core/parser/RustParserDefinition.kt

```diff
@@ -104,6 +104,6 @@ class RustParserDefinition : ParserDefinition {
         /**
          * Should be increased after any change of parser rules
          */
-        const val PARSER_VERSION: Int = LEXER_VERSION + 33
+        const val PARSER_VERSION: Int = LEXER_VERSION + 34
```

Comment:
```suggestion
        const val PARSER_VERSION: Int = LEXER_VERSION + 35
```

`34` version already exists